x-pack/plugins/security_solution/public/management/common/routing.ts

```diff
@@ -118,6 +119,26 @@ const normalizeTrustedAppsPageLocation = (
   }
 };
 
+const normalizeEventFiltersPageLocation = (
```

Comment:
💡  I wonder if we can create a generic utility that could work for any use case in parsing/normalizing url params, so that we did not have to keep cloning this code. maybe a utility that takes in the `defaults` ++ the actual (aka: `location`) value and returns back the location type of information. Likely would be a TS `generic` function so that it can work across use cases.